packages/client/hmi-client/src/assets/css/theme/designer/components/input/_autocomplete.scss

```diff
@@ -56,16 +56,20 @@
         }
     }
 
-    &.p-invalid.p-component > .p-inputtext {
+    &.p-invalid.p-component>.p-inputtext {
         @include invalid-input();
     }
 }
 
 .p-autocomplete-panel {
     background: $inputOverlayBg;
     color: $inputListTextColor;
-    border: $inputOverlayBorder;
-    border-radius: $borderRadius;
+    border: 1px solid var(--surface-border);
+    border-top: none;
+    border-radius: 0;
+    border-bottom-left-radius: 1.5rem;
+    border-bottom-right-radius: 1.5rem;
```

Comment:
Why not use the standard `--border-radius`?

Reply:
Because it wouldn't match the border radius of the search bar. Search input corners are 1.5rem. Autocomplete list should be the same to make a consistent shape